langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/AzureOpenAiStreamingChatModelIT.java

```diff
@@ -234,17 +362,62 @@ public void onError(Throwable error) {
 
         AiMessage aiMessage = response.content();
         assertThat(aiMessage.text()).isNull();
+        List<ChatMessage> messages = new ArrayList<>();
+        messages.add(userMessage);
+        messages.add(aiMessage);
+        assertThat(aiMessage.toolExecutionRequests()).hasSize(3);
```

Comment:
Could you please add a similar test to the `AzureOpenAiChatModelIT`, to make sure that multiple tools at once can be called with sync model as well?

Reply:
Done!